CoreScriptsRoot/Modules/VR/NotificationHub.lua

```diff
@@ -668,6 +697,13 @@ do
 		end
 	end)
 
+	local menuCloseShortcutBindName = game:GetService("HttpService"):GenerateGUID()
```

Comment:
Since these are core actions we do not need to generate random GUIDs for their names. It may be easier in some debugging scenario to have a legitimate name for this binding.